path/matchers.py

```diff
@@ -0,0 +1,74 @@
+from __future__ import absolute_import, unicode_literals
```

Comment:
When you moved the code to another module, did you copy the code byte-for-byte or did you also make changes to it while moving it? A move like this should ideally be done as a multi-phase commit - first copy path.py to path/matchers.py (retaining history), then edit matchers.py to retain only the desired functionality (but not change any behavior), maybe even have an intermediate commit that has `if True:` to keep code at the same level of indentation and reindented, then change any behavior (if necessary). This series of commits will make it clearer to follow the ancestry of commits.